### PR TITLE
4 prepare metaanalyses

### PR DESCRIPTION
**Pull request by wasade**
This includes #164, please do not review/merge until #164 has gone in.

### CODE REVIEW

**Comment by wasade** [comment]
Not clear to me why this failed the first go, but hopefully goes through this time.

**Comment by wasade** [comment]
Unsure what's going on here as the failure is associated with a missing path, however the path appears to exist and is valid [here](https://travis-ci.org/biocore/American-Gut/builds/86738260#L515). Will investigate...

**Comment by wasade** [comment]
Passing. Ready for review

File: ipynb/primary-processing/4-prepare_metaanalyses.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+```python
+>>> original_clean_category_acronym = \
+...     [(ag_md, ag_cleaned_md, 'AGP', 'body_site'),
+...      (gg_md, gg_cleaned_md, 'GG', 'body_site'),
+...      (pgp_md, pgp_cleaned_md, 'PGP', 'body_site'),
+...      (hmp_md, hmp_cleaned_md, 'HMP', 'bodysite')]
+...
+>>> for original, cleaned, acronym, category in original_clean_category_acronym:
+...     with open(original, 'U') as ori_fp, open(cleaned, 'w') as clean_fp:
```
Comment (squirrelo) [inline ipynb/primary-processing/4-prepare_metaanalyses.md]: python lets you do multiple opens in a with? Cool!

**Comment by squirrelo** [comment]
Just a general comment: it's a bit weird to be hiding so much code. I understand it's for the general public so black boxing it is probably for the best, just makes review a bit harder.

**Comment by wasade** [comment]
It's not hidden as the repo is open. Putting the methods into the notebook has the added challenge that it's not possible to unit test those methods. And, the overload of information presented to the user is large.

File: ipynb/primary-processing/4-prepare_metaanalyses.md (hunk 1 above)
Comment (wasade) [inline ipynb/primary-processing/4-prepare_metaanalyses.md]: yup yup :)

**Comment by squirrelo** [comment]
Fair

**Comment by wasade** [comment]
thanks!

**Pull request merged**